src/components/footer/www/footer.jsx

```diff
@@ -11,6 +11,22 @@ const frameless = require('../../../lib/frameless');
 
 require('./footer.scss');
 
+var interwiki_mapping = {
+            'de': 'https://de.scratch-wiki.info',
+            'ru': 'https://ru.scratch-wiki.info/',
+            'nl': 'https://nl.scratch-wiki.info/',
+            'id': 'https://id.scratch-wiki.info/',
+            'ja': 'https://ja.scratch-wiki.info/',
+            'hu': 'https://hu.scratch-wiki.info/',
+            'fr': 'https://fr.scratch-wiki.info/'
+        };
+        var wiki_link = null;
+        if (this.props.intl.locale in interwiki_mapping) {
+            wikilink = interwiki_mapping[this.props.intl.locale];
```

Comment:
Why is there a `this` here? There's no object to access.

Reply:
On line 24 or 25?

Reply:
Both - Travis is saying "Unexpected `this`", and for good reason - you're not in an object's method, so what could `this` possibly refer to?

Reply:
Well, @Kenny2github, I was thinking that it was needed to get the user's language.